### DISCUSSION TITLE
Get Client Config in getServerSideProps

### DISCUSSION THREAD

**Discussion by rajatdhoot123** [vercel/next.js#21859] [Help]
There is some config that is unique per browser (client-side config) I want to access it in getServerSide props is there any way to do that....?

**Comment by BassOfBass** [comment]
Only if you store them in your database and then compare related headers (whatever way you decide to authenticate) to return them as props.

**Reply by rajatdhoot123** [reply]
There is nothing like storing in a database it is just user language preference in which we have to show data

**Reply by BassOfBass** [reply]
The browser language preference is expressed through [`Accept-Language`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language) http header.
